launch/app/pom.xml

```diff
@@ -55,7 +55,14 @@
       <version>${project.version}</version>
       <scope>runtime</scope>
     </dependency>
-    
+
+    <dependency>
```

Comment:
I'd suggest to move the bom dependency up to the other bom dependencies (zwave is the only exception and should be at the end.